client/src/components/Home/Carousel.js

```diff
@@ -1,39 +1,47 @@
-
 import React, { useState } from "react";
 import "../../css/Carousel.css";
-import Carousel from 'react-bootstrap/Carousel'
-import Image from 'react-bootstrap/Image'
+import Carousel from "react-bootstrap/Carousel";
+import Image from "react-bootstrap/Image";
 
-import imageOne from "../../images/image 1.jpg";
+import imageOne from "../../images/image-13.jpg";
 import imageTwo from "../../images/image 2.jpg";
 import imageThree from "../../images/image 3.jpg";
+import { Container, Row,Col } from "react-bootstrap";
```

Comment:
It's best practice to keep external imports (like "react" and "react-bootstrap") at the top of your imports, and have your project's relative imports (like "../../images/image-13.jpg") after.

Also, you can combine all your react-bootstrap imports on one line: 
`import { Container, Row, Col, Carousel, Image } from "react-bootstrap";`